account_move_line_tax_editable/tests/test_account_move_line_tax_editable.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields
+import odoo.tests.common as common
+
+
+class TestAccountMoveLineTaxEditable(common.SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestAccountMoveLineTaxEditable, cls).setUpClass()
+
+        acc_obj = cls.env['account.account']
+        account100 = acc_obj.create({
+            'code': '100',
+            'name': 'Account 100',
+            'user_type_id': cls.env.ref(
```

Comment:
@ernestotejeda  If you need ID then use this `cls.env.('account.data_account_type_receivable')`

Reply:
That doesn't work on `setUpClass` method.